lib/src/super_tooltip.dart

```diff
@@ -329,46 +331,52 @@ class _SuperTooltipState extends State<SuperTooltip>
                 children: <Widget>[
                   Material(
                     color: Colors.transparent,
-                    child: Container(
-                      key: SuperTooltip.bubbleKey,
-                      margin: SuperUtils.getTooltipMargin(
-                        arrowLength: widget.arrowLength,
-                        arrowTipDistance: widget.arrowTipDistance,
-                        closeButtonSize: closeButtonSize,
-                        preferredDirection: preferredDirection,
-                        showCloseButton: showCloseButton,
-                      ),
-                      padding: SuperUtils.getTooltipPadding(
-                        closeButtonSize: closeButtonSize,
-                        showCloseButton: showCloseButton,
-                      ),
-                      decoration: ShapeDecoration(
-                        color: backgroundColor,
-                        shadows: hasShadow
-                            ? <BoxShadow>[
-                                BoxShadow(
-                                  blurRadius: shadowBlurRadius,
-                                  spreadRadius: shadowSpreadRadius,
-                                  color: shadowColor,
-                                ),
-                              ]
-                            : null,
-                        shape: BubbleShape(
-                          arrowBaseWidth: widget.arrowBaseWidth,
+                    child: GestureDetector(
```

Comment:
Thanks for the PR
@ingridwang 
Can you make this optional, not everybody will need this feature

Reply:
The field is optional and defaults to false so nothing will happen on tap. Would you prefer I exclude the GestureDetector if the value is false?